R/MultioutputTask.R

```diff
@@ -0,0 +1,49 @@
+#' @export
+#' @rdname Task
+makeMultioutputTask = function(id = deparse(substitute(data)), data, target, weights = NULL,
+  blocking = NULL, coordinates = NULL, fixup.data = "warn", check.data = TRUE) {
+  assertString(id)
+  assertCharacter(target, any.missing = FALSE, min.len = 2L)
+  assertDataFrame(data)
+  assertChoice(fixup.data, choices = c("no", "quiet", "warn"))
+  assertFlag(check.data)
+
+  ti = which(colnames(data) %in% target)
+  tt = sapply(data[, ti], typeof)
+  if (any(tt == "character")) stop("target data types cannot be character: please convert all characters to factors")
```

Comment:
If you don't mind going to leave some little comments

For these checks I would follow `check.data` and `fixup.data` like in [makeClasifTask](https://github.com/mlr-org/mlr/blob/master/R/ClassifTask.R#L26)